Xamarin.Forms.Platform.Tizen/Renderers/LabelRenderer.cs

```diff
@@ -50,19 +51,31 @@ Native.FormattedString ConvertFormattedText(FormattedString formattedString)
 
 			foreach (var span in formattedString.Spans)
 			{
+				var textDecorations = span.TextDecorations;
+
 				Native.Span nativeSpan = new Native.Span();
 				nativeSpan.Text = span.Text;
 				nativeSpan.FontAttributes = span.FontAttributes;
 				nativeSpan.FontFamily = span.FontFamily;
 				nativeSpan.FontSize = span.FontSize;
 				nativeSpan.ForegroundColor = span.TextColor.ToNative();
 				nativeSpan.BackgroundColor = span.BackgroundColor.ToNative();
+				nativeSpan.Underline = (textDecorations & TextDecorations.Underline) != 0;
+				nativeSpan.Strikethrough = (textDecorations & TextDecorations.Strikethrough) != 0;
 				nativeString.Spans.Add(nativeSpan);
 			}
 
 			return nativeString;
 		}
 
+		void UpdateTextDecorations()
+		{
+			var textDecorations = Element.TextDecorations;
+			Control.Strikethrough = (textDecorations & TextDecorations.Strikethrough) != 0;
```

Comment:
Please, wrap it with Control.BatchBegin() and Control.BatchCommit() for performance like the UpdateFontProperties()
and UpdateFormattedText() no need to call